src/main/java/ru/practicum/shareit/item/storage/ItemInMemoryStorage.java

```diff
@@ -0,0 +1,69 @@
+package ru.practicum.shareit.item.storage;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import ru.practicum.shareit.item.dto.ItemUpdateDto;
+import ru.practicum.shareit.item.model.Item;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+
+@Component
+@Slf4j
+@RequiredArgsConstructor
+public class ItemInMemoryStorage implements ItemStorage {
+
+    private final HashMap<Integer, Item> items = new HashMap<>();
```

Comment:
Лучше обозначать интерфейс, а уже инициализировать объект с определенной реализацией. Так программа становится более гибкой и не сломается, если нужно будет подменить реализацию на другую
```suggestion
    private final Map<Integer, Item> items = new HashMap<>();
```

Reply:
+